src/napari_epitools/_sample_data.py

```diff
@@ -15,6 +15,8 @@
 
 def load_sample_data():
     """Generates an image"""
-    img_path = Path("sample_data") / "8bitDataset" / "test_image.tif"
+    img_path = (
+        Path("sample_data") / "4d" / "181210_DM_CellMOr_subsub_reg_decon-1_cropped.tif"
+    )
```

Comment:
This is the new data they provided?

Reply:
yeah that's the new 4d data - there's only 5 frames in the timeseries so perfect for our needs for now